src/shader_compiler.h

```diff
@@ -53,6 +54,7 @@ class ShaderCompiler {
                         std::vector<uint32_t>* result) const;
 
   std::string spv_env_;
+  bool disable_spirv_validation_;
```

Comment:
Default this to false here.

Reply:
Oops. I have now set it to false in the default constructor. My IDE even warned me about this and I missed it.

Reply:
This is fine for now, for future reference you can do:

```
bool disabled_spirv_validation_ = false;
```
 in the header file which will get set as the default in all constructors (it can still be overridden)